src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudWorkerFailedIT.java

```diff
@@ -63,6 +65,7 @@
 
   @BeforeClass
   public static void init() throws Exception {
+    Assume.assumeFalse(windows);
```

Comment:
Currently, the premise of the ComputeEngineCloudWorkerFailedIT is that worker creation fails because there is no startup script which means Java is not installed on the agent. However, the way that Windows agents are created for integration tests right now means that java comes pre-installed on the image.
Unlike assertFalse, which leads to a failure, assumeFalse will skip tests. We can use this to skip tests it doesn't make sense to run with windows agents like this.